angular-tsconfig.json

```diff
@@ -26,6 +26,7 @@
     "node_modules/@angular/bazel/**",
     "node_modules/@angular/common/upgrade*",
     "node_modules/@angular/compiler-cli/**",
+    "node_moduleS/@angular/dev-infra-private/**",
```

Comment:
Capital S snuck in there

Reply:
good catch. Looks like this still worked like that, but I'm not sure where that came from 😅